src/Shell/Task/TwbsAssetsTask.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+namespace BootstrapUI\Shell\Task;
+
+use Cake\Console\Shell;
+use Cake\Core\Configure;
+use Cake\Core\Plugin;
+use Cake\Filesystem\File;
+use Cake\Filesystem\Folder;
+
+/**
+ * Task for installing Bootstrap assets and layouts to app.
+ */
+class TwbsAssetsTask extends Shell
+{
+    private $_assetDir;
+    private $_nodeDir;
+    private $_cssDir;
+    private $_jsDir;
+
+    /**
+     * TwbsAssetsTask constructor.
+     */
+    public function __construct()
+    {
+        parent::__construct();
+        $this->_assetDir = new Folder(Plugin::path('BootstrapUI') . 'webroot');
+        $this->_nodeDir = new Folder(Plugin::path('BootstrapUI') . 'node_modules');
+        $this->_cssDir = new Folder($this->_assetDir->path . DS . 'css');
+        $this->_jsDir = new Folder($this->_assetDir->path . DS . 'js');
+    }
+
+    /**
+     * Installs Bootstrap assets using npm
+     *
+     * @return void
+     */
+    public function installAssets()
+    {
+        $this->out('<info>Checking npm...</info>');
+        if (!`which npm`) {
+            $this->abort(__('NPM (https://www.npmjs.com/) is required, but not installed. Aborting.'));
```

Comment:
The use of translation function can be removed. It's highly unlike someone is going to spend time translating messages meant for developers.